### PR TITLE
[Np-4240] onKey property of Property

### PR DESCRIPTION
**Pull request by KernelDeimos**

### CODE REVIEW

File: src/foam/u2/tag/Input.js [hunk 1]
```diff
@@ -145,7 +145,8 @@ foam.CLASS({
     function fromProperty(p) {
       this.SUPER(p);
 
-      if ( ! this.hasOwnProperty('onKey') && p.validateObj ) this.onKey = true;
+      if ( ! this.hasOwnProperty('onKey') && (p.validateObj || p.onKey) ) this.onKey = true;
```
Comment (kgrgreer) [inline src/foam/u2/tag/Input.js:148]: What if they explicitly set p.onKey = false? Then it shouldn't be enabled if validatObj is true.

Comment (KernelDeimos) [reply src/foam/u2/tag/Input.js:148]: @kgrgreer something like this should work, right?
```
if ( ! this.hasOwnProperty('onKey') ) this.onKey = p.onKey === undefined ? p.validateObj : p.onKey;
```

Comment (kgrgreer) [reply src/foam/u2/tag/Input.js:148]: if ( this.hasOwnProperty('onKey') ) this.onKey = this.onKey;
else if ( p.hasOwnProperty('onKey') ) this.onKey = p.onKey;
else this.onKey = p.valudateObj;

**Comment by KernelDeimos** [comment]
Closing to use a PR that can be updated: https://github.com/kgrgreer/foam3/pull/446

**Pull request closed without merging**